api/import.go

```diff
@@ -324,22 +324,13 @@ func (h ImportHandler) DownloadCSV(ctx *gin.Context) {
 // Examples:
 //
 // 2,MyApplication,,,,OtherApplication,NORTHBOUND,,,,,,,,
-// 2,MyApplication,OtherApplication,SOUTHBOUND
```

Comment:
This "shorter" way of defining dependencies is not possible anymore, empty columns needs to be used. That should now work as expected (from discussion with Ramon) and fits to CSV template provided in 2.0 release.